### ISSUE TITLE
annotation label 

### ISSUE THREAD

**Issue by uddinnasir** [keras-team/keras-io#1289]
I have one folder in which images and labels are present but my label is in XML file format how can I run this code for my own dataset I need help thanks

**Comment by uddinnasir** [comment]
my xml file formate in this format 




<annotation>
	<folder>dataset</folder>
	<filename>000004.jpg</filename>
	<path>F:\焊接类型\dataset\000004.jpg</path>
	<source>
		<database>Unknown</database>
	</source>
	<size>
		<width>2448</width>
		<height>2048</height>
		<depth>1</depth>
	</size>
	<segmented>0</segmented>
	<object>
		<name>weldinghole</name>
		<pose>Unspecified</pose>
		<truncated>0</truncated>
		<difficult>0</difficult>
		<bndbox>
			<xmin>763</xmin>
			<ymin>455</ymin>
			<xmax>974</xmax>
			<ymax>650</ymax>
		</bndbox>
	</object>
</annotation>

**Comment by sampathweb** [comment]
I would suggest you direct this question to https://discuss.tensorflow.org/ or StackOverflow, where community members might be able to help with some suggestions.
